### DISCUSSION TITLE
Uniswap v4 introduces KYC component: a ripple in the DeFi ecosystem

### DISCUSSION THREAD

**Discussion by serrq** [horizontalsystems/unstoppable-wallet-android#6739] [General]
Bye bye Uniswap, welcome Trader Joe.

https://www.crypto-news-flash.com/uniswap-v4-introduces-kyc-component-a-ripple-in-the-defi-ecosystem/

**Comment by AlekseyFomm** [comment]
exactly))

**Comment by smitmicter049** [comment]
KYC only in v4, v3 without identity verification.

**Comment by Cobra2911** [comment]
right
